WordPress/Classes/ViewRelated/Plans/PlanPostPurchaseViewController.swift

```diff
@@ -328,17 +328,28 @@ class PlanPostPurchasePageViewController: UIViewController {
     }
 }
 
-enum PlanPostPurchasePageType: Int {
-    case PurchaseComplete
-    case Customize
-    case VideoPress
-    case Themes
+enum PlanPostPurchasePageType: String {
+    case PurchaseComplete = "purchase-complete"
+    case Customize = "custom-design"
+    case VideoPress = "videopress"
+    case Themes = "premium-themes"
+    
+    // This is the order we'd like pages to appear in the post purchase flow
+    static var orderedPageTypes: [PlanPostPurchasePageType] { return [ .PurchaseComplete, .Customize, .VideoPress, .Themes ] }
```

Comment:
You can't use `let` for computed properties, and enums can't have stored properties? (you may know something I don't!)